packages/rollup-plugin/src/index.ts

```diff
@@ -84,9 +84,8 @@ export function vanillaExtractPlugin({
       return importsToReplace.reduce((codeResult, importPath) => {
         const assetId = emittedFiles.get(importPath)!;
         const assetName = this.getFileName(assetId);
-        const fixedImportPath = `./${normalize(
-          relative(chunkPath, assetName),
-        )}`;
+        const normalizedImportPath = normalize(relative(chunkPath, assetName)).split(separator).join('/');
```

Comment:
Instead of trying to handle different OS file paths, I think we should just always use posix based paths? I assume Rollup is handling the OS normalization for us. 

You can force posix by instead importing the functions from `path.posix`. @graup thoughts on this?